### PR TITLE
Mismatch in help output and actual arg name

### PR DESCRIPTION
**Pull request by gherceg**
## Technical Summary

Minor typo fix.
### Safety story

Just changing help output text.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/domain/management/commands/hard_delete_forms_and_cases_in_domain.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def handle(self, domain, **options):
         if not options['ignore_domain_in_use'] and is_domain_in_use(domain):
```
Comment (minhaminha) [inline corehq/apps/domain/management/commands/hard_delete_forms_and_cases_in_domain.py:31]: shouldn't it be `options['ignore-domain-in-use-check']`?

**Review by joxl — APPROVED** [review]

File: corehq/apps/domain/management/commands/hard_delete_forms_and_cases_in_domain.py (hunk 1 above)
Comment (gherceg) [reply corehq/apps/domain/management/commands/hard_delete_forms_and_cases_in_domain.py:31]: Reasonable question. If you look at where this argument is defined:
```
        parser.add_argument(
            '--ignore-domain-in-use-check',
            action='store_true',
            dest='ignore_domain_in_use',
            default=False,
            help='Allow deleting forms and cases for a domain that is in use.',
        )
```
You can see that the destination var is specified as `ignore_domain_in_use` 👍🏻

Comment (gherceg) [reply corehq/apps/domain/management/commands/hard_delete_forms_and_cases_in_domain.py:31]: That is still kinda gross that those two vars don't match though. I'll update that.

Comment (gherceg) [reply corehq/apps/domain/management/commands/hard_delete_forms_and_cases_in_domain.py:31]: No worries at all! It lead to fcf2d10e3c542bdf4e1512f208b9aa55aad66f12 which is better (I think)

**Review by minhaminha — APPROVED** [review]

**Review by joxl — APPROVED** [review]

**Pull request merged**